libs/components/lookup/src/lib/modules/lookup/lookup.component.ts

```diff
@@ -567,22 +600,20 @@ export class SkyLookupComponent
       | typeof SkyLookupShowMoreAsyncModalComponent;
 
     if (this.searchAsync.observers.length > 0) {
-      contextProviderType = SkyLookupShowMoreNativePickerAsyncContext;
       modalComponentType = SkyLookupShowMoreAsyncModalComponent;
 
-      contextProviderValue = new SkyLookupShowMoreNativePickerAsyncContext();
+      contextProviderValue = {} as SkyLookupShowMoreNativePickerAsyncContext;
```

Comment:
This won't apply if we keep `SkyLookupShowMoreNativePickerAsyncContext` as a class, but for future reference we should use the object initializer syntax when replacing classes with interfaces:

```
contextProviderValue = {
  idProperty: this.idProperty,
  searchAsync: (args) => {
    this.searchAsync.emit(args);
    return args.result!;
  };
}
```